manual_constraints.txt

```diff
@@ -1,3 +1,2 @@
-sphinx >= 4.5, <6.2.0
```

Comment:
Is this intended?

Reply:
Yes. I have removed some deprecated intersphinx syntax so that using sphinx versions newer than 6.2 does not give deprecation warnings.